src/Tmds.MDns/LinuxHelper.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Net.Sockets;
+
+namespace Tmds
+{
+    static class LinuxHelper
+    {
+        public static unsafe void MultiCastV6(Socket socket, int index)
+        {
+#if NET6_0
+            int setVal = index;
```

Comment:
See comment at the call site, this should no longer be needed.

Reply:
Done